regression/contracts/quantifiers-loop-02/test.desc

```diff
@@ -1,21 +1,22 @@
-KNOWNBUG smt-backend broken-cprover-smt-backend
+CORE
 main.c
---enforce-all-contracts
+--enforce-all-contracts _ --z3
 ^EXIT=0$
 ^SIGNAL=0$
 ^\[main.1\] line .* Check loop invariant before entry: SUCCESS
 ^\[main.2\] line .* Check that loop invariant is preserved: SUCCESS
 ^\[main.assertion.1\] line .* assertion .*: SUCCESS
 ^VERIFICATION SUCCESSFUL$
 --
+^warning: ignoring
 --
 This test case checks the handling of a universal quantifier, with a symbolic
 upper bound, within a loop invariant.
 
-The test is tagged:
-- `smt-backend`:
+TODO: This test should:
```

Comment:
Is there a GitHub issue to track this?

Reply:
Yes I left a comment here, that we should be handling SMT tags in all regression test directories: https://github.com/diffblue/cbmc/issues/5986#issuecomment-845529645